app/models/ncr/work_order.rb

```diff
@@ -114,13 +114,23 @@ def approving_official
       self.approvers.first
     end
 
-    def current_approver_email_address
+    def current_approver
       if self.pending?
-        self.currently_awaiting_approvers.first.email_address
+        self.currently_awaiting_approvers.first
```

Comment:
I agree, I find the `self` a little too selfish. I was just following what appeared to be the convention when I looked at other code in that file, preferring consistency over correctness.